swagger.yaml

```diff
@@ -1116,1114 +1116,1114 @@ definitions:
-                                  properties:
-                                    Amt:
-                                      type: object
-                                      properties:
-                                        Amt:
-                                          type: number
-                                          format: float
-                                          minimum: 0
-                                          example: 50431891779910900
-                                        CCy:
-                                          $ref: '#/definitions/Currencies'
-                                    PyeeFinSvcsPrvdrFee:
-                                      type: object
-                                      properties:
-                                        Amt:
-                                          type: object
-                                          properties:
-                                            Amt:
-                                              type: number
-                                              format: float
-                                              minimum: 0
-                                              example: 50431891779910900
-                                            CCy:
-                                              $ref: '#/definitions/Currencies'
-                                    PyeeFinSvcsPrvdrComssn:
-                                      type: object
-                                      properties:
-                                        Amt:
-                                          type: object
-                                          properties:
-                                            Amt:
-                                              type: number
-                                              format: float
-                                              minimum: 0
-                                              example: 50431891779910900
-                                            CCy:
-                                              $ref: '#/definitions/Currencies'
-            SplmtryData:
-              type: object
-              properties:
-                Envlp:
-                  type: object
-                  properties:
-                    Doc:
-                      type: object
-                      properties:
-                        InitgPty:
-                          type: object
-                          properties:
-                            InitrTp:
-                              type: string
-                              example: 'CONSUMER'
-                            Glctn:
-                              type: object
-                              properties:
-                                Lat:
-                                  type: string
-                                  example: '-3.1291'
-                                Long:
-                                  type: string
+    type: object
```

Comment:
Team, if we change the swagger doc, can we please go to https://editor.swagger.io/ and paste the full document in there and then don't change the formatting after we ensured that it worked on there? That tool is very good metric for swagger docs.

Reply:
Reason I'm asking is I think this indentation will actually break editor.swagger.

Reply:
The indentation doesn't break there.